### PR TITLE
feat: add angular devtools extension

### PR DESCRIPTION
**Pull request by Jefiozie**
With this feature we will add the Angular Devtools to electron, however, because of [#849](https://github.com/rangle/angular-devtools/issues/849) it isn't supported yet.

Making this PR a draft so that we can track the issue and when resolved we can merge it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,7 +42,9 @@
     "@angular-devkit/architect": "0.1200.3",
     "@angular-devkit/core": "^12.0.3",
     "@angular-devkit/schematics": "12.0.3",
-    "electron": "^13.1.1"
+    "electron": "^13.1.1",
+    "electron-devtools-installer": "^3.2.0"
+
```
Comment (bampakoa) [inline package.json:47]:
```suggestion
```

File: src/renderer.ts [hunk 2]
```diff
@@ -11,5 +13,9 @@ function createWindow() {
 }
 
 app.whenReady().then(() => {
+  installExtension(ANGULAR_DEVTOOLS)
+    .then((name) => console.log(`Added Extension:  ${name}`))
+    .catch((err) => console.log('An error occurred: ', err));
+}).then(() => {
```
Comment (bampakoa) [inline src/renderer.ts:19]: Do you know if there is a way to avoid nesting promises?

**Review by bampakoa — COMMENTED** [review]
Thanks for the PR @Jefiozie 🙏 That's awesome! It would be great if you could give some feedback on a couple of comments.

**Comment by bampakoa** [comment]
@Jefiozie can you please resolve the conflicts 🙏 ?

**Comment by Jefiozie** [comment]
Of course!

**Comment by Jefiozie** [comment]
@bampakoa I've updated it based on master and changed the promise to async await.

**Comment by bampakoa** [comment]
@Jefiozie thanks! It looks like a new file has appeared in the PR called `electron-builder.ts`. Is this the same as the `index.ts`?

**Comment by bampakoa** [comment]
@Jefiozie I have updated the `package.json` file. Could you please resolve conflicts and let me know on the previous comment? Thanks 🙏

**Comment by Jefiozie** [comment]
As I think the issue for merging this PR will not be resolved Quickly, what do you think about closing the PR and referring the issue from the devtools in the original issue.

**Comment by bampakoa** [comment]
That's a good idea @Jefiozie 👌 Please, go ahead.

**Comment by bampakoa** [comment]
@Jefiozie in the latest version of the extension, I have tested it, and works as expected. I have filed a relevant issue here #12 if you are still interested in submitting a new PR 😃

**Pull request closed without merging**